iOS Client/DevotionsController/DevotionTableViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DevotionTableViewCell.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 11/22/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import UIKit
+
+class DevotionTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var dayLabel: UILabel!
+    @IBOutlet weak var monthLabel: UILabel!
+    @IBOutlet weak var verseLabel: UILabel!
+    @IBOutlet weak var readNotifier: UIImageView!
+    
+    
+    func configure(day: String, month: String, verse: String, read: Bool) {
```

Comment:
a minor note, but this could just accept one parameter, the LoadedDevotion

Reply:
nice I like that